docs/getting-started/release-notes/v2_4_0.md

```diff
@@ -82,21 +86,16 @@ See [New features and enhancements](#new-features-and-enhancements) for a full l
 
 #### Zowe CLI (Core)
 
-#### Zowe CLI Imperative Framework
-
-
-
-#### Db2 Plug-in for Zowe CLI
-
-
-### Zowe Explorer
-
-
-
-
-#### Zowe Explorer Extension for FTP
+- Updated the `minimatch` and `keytar` dependencies for technical currency.
```

Comment:
personal opinion: I believe that dependency updates should always be at the bottom of all lists as they may not impact end users too much (unless there is something seriously broken that we want to highlight with that dependency update).
This comment applies to the imperative framework dependency updates too.

Reply:
sounds like a good practice to me, thanks for letting me know : )